api/v1/endpoints/get-identity-map.md

```diff
@@ -18,13 +18,15 @@ Retrieve the UID2 and salt bucket ID for an email address or an email address ha
 
 ###  Query Parameters
 
-* Only one of the following two query parameters is required. 
+* Only one of the following four query parameters is required. 
 * If both parameters are included in a request, only the `email` will return a response.
 
 | Query Parameter | Data Type | Attribute | Description |
 | :--- | :--- | :--- | :--- |
 | `email` | string | Conditionally Required | The email address to be mapped. |
 | `email_hash` | string | Conditionally Required | The [URL-encoded, base64-encoded SHA256](../../README.md#email-address-hash-encoding) hash of the [normalized](../../README.md#email-address-normalization) email address. |
+| `phone` | string | Conditionally Required | The [normalized](../../README.md#phone-number-normalization) phone number for which to generate tokens. |
```

Comment:
Given phone numbers start with a plus character, I think we should explicitly state that the phone number must be URL-encoded. Would also be beneficial to add similar note for raw email address above.

Reply:
Updated all endpoints, except POST /identity/map, which doesn't seem to be using URL-encoded values, based on the provided examples.